keyboards/keebio/bfo9000/keymaps/abstractkb/keymap.c

```diff
@@ -0,0 +1,98 @@
+#include QMK_KEYBOARD_H
+
+// LAYER NAME MACROSS
+#define _BASE 0
+#define _LIST 1
+
+// Fillers to make layering more clear
```

Comment:
QMK defines these keycode aliases automatically. Lines 7-9 should be deleted.

https://github.com/qmk/qmk_firmware/blob/a120a000d481ed14a87988383be46c5a76604bb6/quantum/quantum_keycodes.h#L29-L31

```suggestion
```